NOTICE

```diff
@@ -6,3 +6,6 @@ Boxever Ltd. (http://www.boxever.com/).
 
 This product includes software developed at
 SoundCloud Ltd. (http://soundcloud.com/).
+
+This product includes software developed at
+Netflix Inc. (https://www.netflix.com).
```

Comment:
I'd specify which repo it was copied from, as the rest were developed in this repo.